app/fissile.go

```diff
@@ -951,6 +951,14 @@ func (f *Fissile) generateHelmValues(outputDir string, rolesManifest *model.Role
 		_, err = outputFile.Write(buf)
 	}
 	if err == nil {
+		_, err = outputFile.Write([]byte(`kube:
```

Comment:
Not a fan of embedding YAML in the source like this. (I'm less against doing it in tests, but this isn't that.)

Is there something we can do here better? Put it in `buf` next to `env`?